src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsdocumentmanagementapi/repository/DocumentRepositoryIntTest.kt

```diff
@@ -0,0 +1,142 @@
+package uk.gov.justice.digital.hmpps.hmppsdocumentmanagementapi.repository
+
+import io.hypersistence.utils.hibernate.type.json.internal.JacksonUtil
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.assertj.core.api.Assertions.within
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.test.context.jdbc.Sql
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.hmppsdocumentmanagementapi.entity.Document
+import uk.gov.justice.digital.hmpps.hmppsdocumentmanagementapi.enumeration.DocumentType
+import uk.gov.justice.digital.hmpps.hmppsdocumentmanagementapi.integration.IntegrationTestBase
+import java.time.LocalDateTime
+import java.time.temporal.ChronoUnit
+import java.util.UUID
+
+@Transactional
+class DocumentRepositoryIntTest : IntegrationTestBase() {
```

Comment:
These tests will likely be replaced by service integration tests when the full service stack is in place